### PR TITLE
MaskBTP adds property for masking named components

### PR DESCRIPTION
**Pull request by peterfpeterson**
This is intended for masking the front or back planes of EQ-SANS, but it
written to work generically. Using this property also makes the
algorithm work with any instrument in this mode.

**To test:**

Try it out on the (in progress) EQ-SANS IDF.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/algorithms/MaskBTP-v1.rst [hunk 1]
```diff
@@ -10,27 +10,27 @@
...
-applies to the following instruments only: ARCS, CNCS, CORELLI, HYSPEC, NOMAD,
-POWGEN, SEQUOIA, SNAP, TOPAZ. For instruments with rectangular position
-sensitive detectors (POWGEN, SNAP, TOPAZ), the tube is corresponding to
-the x coordinate, and pixel to the y coordinate. For example, on SNAP
-Bank="1", Tube="3" corresponds to 'SNAP/East/Column1/bank1/bank1(x=3)',
-and Bank="1", Tube="3", Pixel="5" is 'SNAP/East/Column1/bank1/bank1(x=3)/bank1(3,5)'.
-
-If one of Bank, Tube, Pixel entries is left blank, it will apply to all
+Algorithm to mask detectors in particular banks, tube, or pixels.
+To use the ``Bank`` argument, the instrument must be one of: ARCS, BIOSANS, CG2 CNCS, CORELLI, EQ-SANS HYSPEC, MANDI, NOMAD, POWGEN, SEQUOIA, SNAP, SXD, TOPAZ, WAND, WISH.
```
Comment (jmborr) [inline docs/source/algorithms/MaskBTP-v1.rst]: Instrument scientist (and me :smile: ) prefer using string `EQSANS` instead of `EQ-SANS`.

**Comment by jmborr** [comment]
File I'll be using for testing 
[EQSANS_Definition_20190615_21001231.xml.zip](https://github.com/mantidproject/mantid/files/3299179/EQSANS_Definition_20190615_21001231.xml.zip)

File: docs/source/algorithms/MaskBTP-v1.rst (hunk 1 above)
Comment (peterfpeterson) [reply docs/source/algorithms/MaskBTP-v1.rst]: Either will work in the `InstrumentName` parameter.

**Review by jmborr — CHANGES_REQUESTED** [review]
We can leave the EQSANS name hurdle for latter, but can you fix the `sans.rst` conflicts?

**Review by jmborr — APPROVED** [review]

**Comment by jmborr** [comment]
should be merged after the remaining jenkin server passes

**Pull request merged**